tests/scripts/thread-cert/Cert_8_1_01_Commissioning.py

```diff
@@ -73,70 +93,171 @@ def test(self):
+        #    a DTLS-ChangeCipherSpec record and
+        #    an encrypted DTLS-Finished handshake record to the Commissioner.
+        pkts.filter_wpan_dst64(COMMISSIONER).\
+            filter(lambda p:
+                       p.dtls.handshake.type == [HANDSHAKE_CLIENT_KEY_EXCHANGE] and\
+                       {
+                        CONTENT_CHANGE_CIPHER_SPEC,
+                        CONTENT_HANDSHAKE
+                       } == set(p.dtls.record.content_type)
+                       ).\
+               must_next()
 
-        # TODO(wgtdkp): It's required to verify DTLS FINISHED message here.
-        # Currently not handled as it is encrypted.
+        # 7. Commissioner receives the DTLS-ClientKeyExchange handshake record, the
+        #    DTLS-ChangeCipherSpec record and the encrypted DTLS-Finished handshake record,
+        #    and sends a DTLS-ChangeCipherSpec record and an encrypted DTLSFinished handshake
+        #    record in that order to Joiner
+        pkts.filter_wpan_src64(COMMISSIONER).\
+            filter(lambda p: {
+                              CONTENT_CHANGE_CIPHER_SPEC,
+                              CONTENT_HANDSHAKE
+                             } == set(p.dtls.record.content_type)
+                      ).\
+              must_next()
 
-        # 5.8,9,10,11
-        # - Joiner_1
-        command.check_joiner_commissioning_messages(joiner_messages.commissioning_messages)
-        # - Commissioner
-        command.check_commissioner_commissioning_messages(commissioner_messages.commissioning_messages)
-        # As commissioner is also joiner router
-        command.check_joiner_router_commissioning_messages(commissioner_messages.commissioning_messages)
+        # 8. Joiner receives the DTLS-ChangeCipherSpec record and the encrypted DTLS-Finished
+        #    handshake record and sends a JOIN_FIN.req message in an encrypted DTLS-ApplicationData
+        #    record in a single UDP datagram to Commissioner.
+        pkts.filter_wpan_dst64(COMMISSIONER).\
+            filter(lambda p:
+                   [CONTENT_APPLICATION_DATA] == p.dtls.record.content_type
+                  ).\
+               must_next()
 
-        self.nodes[JOINER].thread_start()
-        self.simulator.go(5)
-        self.assertEqual(self.nodes[JOINER].get_state(), 'router')
+        # 9. Commissioner receives the encrypted DTLS-ApplicationData record and sends a
+        #    JOIN_FIN.rsp message in an encrypted DTLS-ApplicationData record in a single
+        #    UDP datagram to Joiner
+        pkts.filter_wpan_src64(COMMISSIONER).\
+            filter(lambda p:
+                   [CONTENT_APPLICATION_DATA] == p.dtls.record.content_type
+                  ).\
+               must_next()
+
+        # 10. Commissioner sends an encrypted JOIN_ENT.ntf message to Joiner
+
+        # 11. Joiner receives the encrypted JOIN_ENT.ntf message and sends an encrypted
+        #     JOIN_ENT.ntf dummy response to Commissioner
```

Comment:
`# 12. Joiner_1 sends an encrypted DTLS-Alert record with a code of 0 (close_notify) to Commissioner` 
is missed

Reply:
added the step